dashboard/db/migrate/20160715015742_set_type_for_pd_workshop_sections.rb

```diff
@@ -0,0 +1,13 @@
+class SetTypeForPdWorkshopSections < ActiveRecord::Migration
+  def up
+    Pd::Workshop.where.not(section:nil).each do |workshop|
```

Comment:
Though it won't matter (seeing as there are so few sections), I would encourage using `find_each`, both for good form and in case this code gets copied-and-pasted. Also below.